packages/app/app/components/HLSPlayer/index.tsx

```diff
@@ -0,0 +1,171 @@
+import React, {RefObject, ReactElement} from 'react';
+import ReactHlsPlayer from 'react-hls-player';
+
+enum SoundStatus {
+  PAUSED = 'PAUSED',
+  PLAYING = 'PLAYING',
+  STOPPED = 'STOPPED',
+}
+
+export enum SoundErrors {
```

Comment:
These errors should be localized and kept in the locale files.

Reply:
I did the error same as `react-hifi` and also `handleError` doesn't display the error message
```js
  handleError(err) {
    logger.error(err.message);
    this.props.actions.streamFailed();
  }

```

do you want to `toast` the error message it will also work on `Sound` component of `react-hifi`

Reply:
Ok, I guess that's fine then